lib/SetVarMainTemplatePlugin.js

```diff
@@ -4,33 +4,46 @@
 */
 var ConcatSource = require("webpack-sources").ConcatSource;
 
-function SetVarMainTemplatePlugin(varExpression, copyObject) {
-	this.varExpression = varExpression;
+function SetVarMainTemplatePlugin(varExpressions, copyObject) {
+	if (varExpressions && varExpressions.constructor !== Array) {
+		varExpressions = [varExpressions];
+	}
+	this.varExpressions = varExpressions.length > 0 ? varExpressions : undefined;
 	this.copyObject = copyObject;
 }
 module.exports = SetVarMainTemplatePlugin;
 SetVarMainTemplatePlugin.prototype.apply = function(compilation) {
 	var mainTemplate = compilation.mainTemplate;
 	compilation.templatesPlugin("render-with-entry", function(source, chunk, hash) {
-		var varExpression = mainTemplate.applyPluginsWaterfall("asset-path", this.varExpression, {
-			hash: hash,
-			chunk: chunk
+		var varExpressionsSrc = this.varExpressions.map(function(varExpressionSrc) {
+			return mainTemplate.applyPluginsWaterfall("asset-path", varExpressionSrc, {
+				hash: hash,
+				chunk: chunk
+			});
 		});
 		if(this.copyObject) {
-			return new ConcatSource("(function(e, a) { for(var i in a) e[i] = a[i]; }(" +
-				varExpression + ", ", source, "))");
+			return new ConcatSource("(function(e, a) { for(var i in a) " +
+					"for(var j=0; j<e.length; j++) e[j][i] = a[i];" +
```

Comment:
What do you mean with "spacing"?

Yep will update to output original if not multi library.